src/EurekaClient.js

```diff
@@ -303,19 +303,21 @@ export default class Eureka {
    */
   transformRegistry(registry) {
     if (!registry) {
-      throw new Error('Unable to transform empty registry');
-    }
-    this.cache = { app: {}, vip: {} };
```

Comment:
I changed it because I thought I saw this today while running getInstancesByVIp in a loop. Hmm. But, I see what you're saying. Thoughts?